spec/graphql/schema/directive/flagged_spec.rb

```diff
@@ -5,6 +5,10 @@
   class FlaggedSchema < GraphQL::Schema
     module Animal
       include GraphQL::Schema::Interface
+      if GraphQL::Schema.use_schema_subset?
+        # It won't check possible types, so it needs this directly
+        directive GraphQL::Schema::Directive::Flagged, by: ["northPole", "southPole"]
+      end
```

Comment:
With `Warden`, this would be hidden when all possible types were hidden. But Subset doesn't check that, so I added a visibility rule that would do the same thing.